src/Env/Dumper.php

```diff
@@ -33,8 +34,15 @@ public static function dump(Env $env, $workDir)
     {
         $fs = new Filesystem();
 
+        $export = $env->export();
+        $fs->dumpFile("$workDir/ansible/.manala.yml", Yaml::dump([
+            'envs' => [
```

Comment:
JFYI: I didn't handled the case we have a project with multiple environments inside, because it is not supported right now. But the metadata file structure needs to plan ahead this.